infra/main.bicep

```diff
@@ -126,6 +126,22 @@ resource keyVault 'Microsoft.KeyVault/vaults@2022-07-01' = {
   }
 }
 
+resource keyVaultDiagnosticLogs 'Microsoft.Insights/diagnosticSettings@2021-05-01-preview' = {
+  name: keyVault.name
+  scope: keyVault
+  properties: {
+    workspaceId: logAnalyticsWorkspace.id
+    logs: [for logCategory in [ 'AuditEvent' ]: {
```

Comment:
why is this a loop?
if we might want to add more categories, shouldn't we move the logCategories to a variable?

Reply:
I'll change it.